README.md

```diff
@@ -54,46 +54,48 @@ Currently, [Azure Key Vault plugin for Notation](https://github.com/Azure/notati
     target_artifact_reference: <list_of_target_artifact_references_in_remote_registry>
     signature_format: <signature_envelope_format>
     plugin_config: <list_of_plugin_defined_configs>
-    allow_referrers_api: <boolean_flag_for_referrers_api>
+    force_referrers_tag: <boolean_flag_for_referrers_tag_schema>
+    timestamp_url: <url_of_RFC_3161_Timestamp_Authority_server>
+    timestamp_root_cert: <filepath_of_RFC_3161_Timestamp_Authority_root_certificate>
 ```
 
 <details>
 
 <summary>See an example (Click here).</summary>
 
 ```yaml
-- name: sign releasd artifact with notation-azure-kv plugin
+- name: sign releasd artifact with notation-azure-kv plugin and timestamping
   uses: notaryproject/notation-action/sign@v1
   with:
     plugin_name: azure-kv
     plugin_url: https://github.com/Azure/notation-azure-kv/releases/download/v1.2.0/notation-azure-kv_1.2.0_linux_amd64.tar.gz
     plugin_checksum: 06bb5198af31ce11b08c4557ae4c2cbfb09878dfa6b637b7407ebc2d57b87b34
     key_id: https://testnotationakv.vault.azure.net/keys/notationLeafCert/c585b8ad8fc542b28e41e555d9b3a1fd
     target_artifact_reference: |-
-      myRegistry.azurecr.io/myRepo@sha256:aaabbb
-      myOtherRegistry.azurecr.io/myOtherRepo@sha256:cccddd
+      myregistry.azurecr.io/myrepo@sha256:b94d27b9934d3e08a52e52d7da7dabfac484efe37a5380ee9088f7ace2efcde9
+      myotherregistry.azurecr.io/myotherrepo@sha256:aaad27b9934d3e08a52e52d7da7dabfac484efe37a5380ee9088f7ace2efcaaa
     signature_format: cose
     plugin_config: |-
       ca_certs=.github/cert-bundle/cert-bundle.crt
       self_signed=false
+    timestamp_url: http://myTrustedTimestmapAuthority.com
```

Comment:
`myTrustedTimestmapAuthority.com` can be a real host. Maybe we can try the fake one from the notary spec
```suggestion
    timestamp_url: http://my.trusted.timestmap.authority.wabbit-networks.io
```

Reply:
Good idea. Updated.